test/test_exodus.py

```diff
@@ -23,7 +23,7 @@ def test_read_exodus(self):
     def test_init_verts(self):
         """Create a uxarray grid from vertices and saves a 1 face exodus
         file."""
-        verts = np.array([[0, 0], [2, 0], [0, 2], [2, 2]])
+        verts = [[[0, 0], [2, 0], [0, 2], [2, 2]]]
```

Comment:
Any particular reason for changing this?

Reply:
Ideally, we still want the user to pass in the one face vertexes in this way `[[[0, 0], [2, 0], [0, 2], [2, 2]]]` instead of just the `meshface_nodes * coord` since that's should be the correct format.

I supported the `meshface_nodes * coord` input just because the origial `__from_vert__()` did that, and forbiding this input type will require more underlying changes in other functions if such functions calls `__from_vert__()`

Reply:
Makes sense!